src/api/spec/jobs/report_to_scm_job_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe ReportToScmJob, vcr: false do
```

Comment:
Is `vcr` set to `false` on purpose? Could it just be removed?

Reply:
Enabling this just records calls to backend that are not related to the spec. So I think we don't need to enable this.

Reply:
so no need to add the flag, right?

Reply:
I'm working on an improved spec, I will take care of that there